charts/lepus/values.yaml

```diff
@@ -0,0 +1,180 @@
+  # ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  livenessProbe:
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 30
+  readinessProbe:
+    initialDelaySeconds: 31
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 30
+
+## pod autoscaling configuration, use for automatic scaling of the broker pods
+autoscaling:
+  #  -- should autoscaling be enabled for lepus
+  enabled: false
+  # -- minimum number of running pods
+  minReplicas: 1
+  # -- maximum number of running pods
+  maxReplicas: 10
+  # -- metrics to react on
+  metrics: []
+  ## List of MetricSpecs to decide whether to scale
+  # See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#metricspec-v2beta2-autoscaling
+  # scaler targets to hold average cpu around 80%
+  #- type: Resource
+  #  resource:
+  #    name: cpu
+  #     target:
+  #      type: Utilization
+  #      averageUtilization: 80
+  ## scaler targets to hold average memory around 80%
+  #  - type: Resource
+  #    resource:
+  #      name: memory
+  #      target:
+  #        type: Utilization
+  #        averageUtilization: 80
+
+## openshift specific route definition. Will not work on plain k8s
+route:
+  ## -- should the deployment create openshift routes
+  enabled: false
+  # -- annotations to be added to the route
+  annotations: {}
+  # -- host to be used
+  # host: localhost
+  # -- tls configuration for the route
+  tls: {}
+  # termination: edge
+
+## ingress configuration
+ingress:
+  # -- should there be an ingress to connect lepus with the public internet
+  enabled: false
+  # -- annotations to be added to the ingress
+  annotations: {}
+    # kubernetes.io/ingress.class: "ambassador"
+    ## example annotations, allowing cert-manager to automatically create tls-certs and forcing everything to use ssl.
+    # kubernetes.io/tls-acme: "true"
+    # ingress.kubernetes.io/ssl-redirect: "true"
+  # -- all hosts to be provided
+  hosts: []
+    ## provide a hosts and the paths that should be available
+    # - host: localhost
+      # paths:
+      # - /
+  # -- configure the ingress' tls
+  tls: []
+    # - secretName: lepus-tls
+      # hosts:
+        # - lepus.fiware.org
+
+# -- port that the lepus container uses
+port: 3000
+# -- host where lepus is available at
+host: http://localhost
+## Regex for for debug output
+debug: adapter:*
+
+userContext: 
+  # for the fixed JSON-LD @context to be supplied
+  url: https://fiware.github.io/tutorials.Step-by-Step/tutorials-context.jsonld
+coreContext:
+  #  or the fixed core NGSI-LD @context to be supplied 
+  url: https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.9.jsonld
+lepus:
+  # host alias for Lepus
+  alias: lepus
+  # maximum length of time to forward the NGSI-LD notification
+  timeout: 1000
+  #  for the location of Lepus itself
+  url: https://adapter:3000
+  # whether to use multiple cores
+  multiCore: true
```

Comment:
Wouldn't it be better to decide the use of multicore based on the resource sets?